### DISCUSSION TITLE
Telegram shows only generic icon

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2007] [General]
I don't open an issue because I cannot reproduce it or nail it down, but it happened some days ago on debian+git and now I rebooted arch and it's there too.
* it's not due to a  telegram update
* changing icon theme  doesn't help
* tested with xfce4-panel, telegram icon is ok
* on arch it got fixed by changing in panel settings icon size down to 10 and back to 40 (clicking the triangles), on debian this doesn't help.

It's quite annoying because the icon notifies about new messages.
Btw a new  category "problem solving" or similar?

**Comment by tsujan** [comment]
Not reproducible here. I use Telegram a few times a day.

Make sure you use Status Notifier and not the obsolete system tray plugin.

**Reply by stefonarch** [reply]
Sure I don't use the tray plugin and  as said I cannot nail it down to something precise, but it's weird that it happened on both machines now.

**Reply by tsujan** [reply]
We've had only 11 commits since lxqt-panel 0.17.1 and none of them is about Status Notifier. I don't think that lxqt-panel could cause the problem you see.

Maybe it's about a change that you made on both systems, like using the same icon theme.

**Comment by stefonarch** [comment]
Icon theme is Papirus, and I tried every possible alternative in panel settings. 
I tested now a clean user on debian and the issue didn't appear, and logging back in it was gone too... So no idea what the reason was for not drawing the right icon.

**Reply by tsujan** [reply]
> I tested now a clean user on debian and the issue didn't appear, and logging back in it was gone too

Something in Telegram's config directory?

**Reply by stefonarch** [reply]
No idea, I don't think, restarted panel and telegram many times. If it happens again maybe something can be found.
